canonical-web-and-design/global-nav
# Add run script to the project

## Done
Set up `./run` on the project.
Updated the README
Added [http-server](https://www.npmjs.com/package/http-server) to serve the demo page locally

## QA
- Run `./run clean` and check the dist and node_modules folders are removed
- Run `./run build` and check the dist folder is created
- Run `./run serve` then open http://0.0.0.0:8300/ and see the demo
- Run `./run test` and check the tests pass
- Check the README reads ok
